tests/unit/pages/categories/Categories.spec.jsx

```diff
@@ -0,0 +1,38 @@
+import '@testing-library/jest-dom'
+import { renderWithProviders } from '~tests/test-utils'
+import { screen } from '@testing-library/react'
+import Categories from '~/pages/categories/Categories'
+import { vi } from 'vitest'
+
+vi.mock('~/components/page-wrapper/PageWrapper', () => {
+  return {
+    __esModule: true,
+    default: ({ children }) => <div>{children}</div>
+  }
+})
+
+vi.mock('~/context/modal-context', async () => {
```

Comment:
You don`t need mock context, if you have mocked all of PageWrapper, because you just have replaced this component to div with children, I guess.

Reply:
Yes, I suppose it's logical. Thanks for pointing it out